source/speech/speech.py

```diff
@@ -78,12 +79,31 @@
 _curWordChars: List[str] = []
 
 
-class SpeechMode(IntEnum):
+class SpeechMode(DisplayStringIntEnum):
 	off = 0
 	beeps = 1
 	talk = 2
 	onDemand = 3

```

Comment:
This will be back compat breaking, and I am not sure if renaming these members just to comply with our coding standards is worth it. I am also not sure how difficult it will be to preserve backwards compatibility here - enum's use various meta programming tricks, so implementing custom `__getattr__` may not be trivial. I will look into this if you still think its important though. Alternatively we can rename the on demand mode using screaming snake case, so that it is clear that new members should be named that way going forward. This can be done safely, as this mode was introduced in the current development cycle.

Reply:
My mistake - I thought this came from #15804